casanovo/casanovo.py

```
@@ -694,17 +698,33 @@ def _get_weights_from_url(
             file_response = requests.head(file_url)
             if file_response.ok:
                 if "Last-Modified" in file_response.headers:
-                    url_last_modified = datetime.datetime.strptime(
-                        file_response.headers["Last-Modified"],
-                        "%a, %d %b %Y %H:%M:%S %Z",
-                    ).timestamp()
+                    try:
+                        url_last_modified = datetime.datetime.strptime(
+                            file_response.headers["Last-Modified"],
+                            "%a, %d %b %Y %H:%M:%S %Z",
+                        ).timestamp()
```

Ruff (0.15.9)

[warning] Naive datetime constructed using `datetime.datetime.strptime()` without %z

(DTZ007)